src/main/java/com/odysseusinc/arachne/portal/service/impl/BaseDataSourceServiceImpl.java

```diff
@@ -212,6 +217,16 @@ public DS updateInAnyTenant(DS dataSource)
         return savedDataSource;
     }
 
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @Override
+    public DS updateDataSourceTenants(final DS dataSource) 
```

Comment:
Why do you put the method in community, if it is used only in enterprise?

Reply:
done